warehouse/query-core/src/test/java/datawave/query/tables/facets/FacetedQueryLogicTest.java

```diff
@@ -164,7 +163,6 @@ public void testQueryDynamicFacets() throws Exception {
         // @formatter:off
         expected.add("null; paris -- paris//1");
         expected.add("null; rome -- rome//2");
-        // @formatter:on
```

Comment:
this seems like a mistake.

Reply:
Yup, good catch. Fixed in the next commit.